client/my-sites/plugins/index.js

```diff
@@ -102,6 +103,9 @@ export default function () {
 			clientRender
 		);
 
+		if ( config.isEnabled( 'marketplace-yoast' ) ) {
+			page( '/plugins/domain/:site?', siteSelection, renderDomainsPage, makeLayout, clientRender );
```

Comment:
When the `:site` parameter is optional, then `/plugins/domain` is also a valid route. Does it make sense to have this UI without a selected site?

When I try to access `/plugins/domain` locally, I get an error notice, probably caused by adding a Yoast product into a non-existent cart:

<img width="506" alt="Screenshot 2021-05-06 at 9 13 59" src="https://user-images.githubusercontent.com/664258/117260790-bbb85000-ae4f-11eb-8930-42efc1cc5c76.png">

Reply:
Thanks It had not struck me properly what that question mark meant :) Will remove it!